gravity.sh

```diff
@@ -43,7 +45,7 @@ else
 fi
 
 #Remove the /* from the end of the IPv4addr.
-IPv4addr=${IPv4_address%/*}
+IPv4_address=${IPv4_address%/*}
```

Comment:
on install: should not break installer script as the info for setDHCPCD() is gatherd from find_IPv4_information() 
on update/reconfigure:even if i could find where functions were called from these scripts. ${IPv4_address} (from setupVars.conf) would be overwriten by find_IPv4_information()